### PR TITLE
android-studio: update to 2022.3.1.18, orphan

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **Not sure, template already exist** 

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

This PR is based on #41215

### CODE REVIEW

**Comment by Luciogi** [comment]
I have doubts:
- This package has optional dependency on ncurses5, though application works without it. [AUR package points that it required for native debugger](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=android-studio#n23)
- Please tell is `nopie` option is required [here](https://github.com/void-linux/void-packages/pull/45676/files#diff-76718ef6b79845dc9aa817671a5390627e1d108501cc5d4dbd07c002fe4822d0L22)
---
For maintainer ship , I m not fully ready

File: srcpkgs/android-studio/template [hunk 1]
```diff
@@ -1,83 +1,72 @@
...
+/opt/android-studio/plugins/android/resources/perfetto/armeabi-v7a/traced
+/opt/android-studio/plugins/android/resources/perfetto/armeabi-v7a/perfetto
+/opt/android-studio/plugins/android/resources/perfetto/arm64-v8a/libperfetto.so
+/opt/android-studio/plugins/android/resources/perfetto/arm64-v8a/traced_probes
+/opt/android-studio/plugins/android/resources/perfetto/arm64-v8a/traced
+/opt/android-studio/plugins/android/resources/perfetto/arm64-v8a/perfetto
+/opt/android-studio/plugins/android/resources/perfetto/x86/libperfetto.so
+/opt/android-studio/plugins/android/resources/perfetto/x86/traced_probes
+/opt/android-studio/plugins/android/resources/perfetto/x86/traced
+/opt/android-studio/plugins/android/resources/perfetto/x86/perfetto"
```
Comment (Bnyro) [inline srcpkgs/android-studio/template:57]: Please indent these lines

**Comment by Bnyro** [comment]
As described [here](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#orphaning-a-template), please add `, orphan` to your commit message.

**Comment by Bnyro** [comment]
> This package has optional dependency on ncurses5, though application works without it
>
Optional dependencies should likely only be included if they're required for some fundamental features of the package - and I don't think that's the case here.

> Please tell is nopie option is required [here](https://github.com/void-linux/void-packages/pull/45676/files#diff-76718ef6b79845dc9aa817671a5390627e1d108501cc5d4dbd07c002fe4822d0L22)
>
It seems to build just fine without using nopie, so I think it can be removed here.

**Comment by Luciogi** [comment]
@Bnyro  what about common/shlibs?

**Comment by Bnyro** [comment]
I'm not really familiar with that, sorry.

Looks fine to me, but you should rely on me for that one.

**Comment by Luciogi** [comment]
can you ping someone to review ?

**Comment by Bnyro** [comment]
You can ask in the IRC room, it's linked in the Readme.

**Comment by Bnyro** [comment]
Should be https://web.libera.chat/?nick=Guest?#xbps

File: srcpkgs/android-studio/template [hunk 2]
```diff
@@ -1,83 +1,71 @@
...
-# https://developer.android.com/studio/#resources as of 2018-07-12
-_studio_build=201.7199119
-_studio_rev=0
-archs="x86_64 i686"
+archs="x86_64"
 hostmakedepends="tar"
-depends="virtual?libGL"
 short_desc="Official Android IDE"
-maintainer="Jordyn Carattini <onlinecloud1@gmail.com>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ahesford) [inline srcpkgs/android-studio/template:8]: Why are you orphaning a package that you don't maintain?

Comment (Luciogi) [reply srcpkgs/android-studio/template:8]: package have not updated from long time

Comment (Luciogi) [reply srcpkgs/android-studio/template:8]: should I revert it ?

Comment (ahesford) [reply srcpkgs/android-studio/template:8]: Looks like the maintainer hasn't contributed to Void in several years, so we can keep the orphan change.

**Comment by Luciogi** [comment]
@ahesford  can you tell me about `common/shlibs` I dont understand which libs to include

**Comment by ahesford** [comment]
No libraries from this package should be mentioned in 'common/shlibs`. Please remove your additions as well as the existing entries that you intended to replace.

**Comment by Luciogi** [comment]
@ahesford  after deleting libs from `common/shlibs` breaks package build
https://pastebin.com/Ug5xKLHT

**Comment by ahesford** [comment]
The inclusion of ncurses v5 compact pins has been discussed and rejected in the past. Skipping dependency checks for those two libraries is one possible workaround, but it may be better to remove those CPython extensions entirely (and anything that depends on them) because they will not be importable by Python.

**Comment by Luciogi** [comment]
@ahesford , Lint template test fails

**Comment by Bnyro** [comment]
`authored by noreply email` - you have to use a proper email for authoring the commit.

**Comment by Luciogi** [comment]
> 

would simplelogin alias work here?

**Comment by Bnyro** [comment]
You should use an email you have and keep access to, I'm not familiar with Simplelogin.

**Comment by Luciogi** [comment]
lol same test was passed just before this commit

**Comment by Bnyro** [comment]
Only a network error, can be ignored.

**Comment by Luciogi** [comment]
> Only a network error, can be ignored.

should I force push again
or reviewer will manually start checks ?

**Pull request merged**